Cargo.toml

```diff
@@ -96,7 +96,7 @@ bitflags = "2.4"
 bitstruct = "0.1"
 bitvec = "1.0"
 byteorder = "1"
-bytes = "1.1"
+bytes = "1.7.1"
```

Comment:
Note: This matches omicron/main, as I ran into issues when using omicron/main patched without this update.